.github/workflows/build_test.yaml

```diff
@@ -0,0 +1,32 @@
+name: Build Test
+on: [push]
+
+jobs:
+  build:
+    name: Validate Changes
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [2.7, 3.8]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Fetch master (to enable diff)
+      run: |
+        git fetch origin master
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip setuptools wheel
+        pip install PyYAML argparse
+        pip install catkin-pkg ros-buildfarm rosdistro nose coverage
+        pip install yamllint
+        pip install unidiff
+        pip install rosdep
+        pip install PyGithub
+    - name: Run yamllint
+      run: yamllint */
+    - name: Run Nose Tests
+      run: nosetests -s
```

Comment:
This is probably just my unfamiliarity with Github actions/rosdistro tests, but how does just fetching the master data work?  Do the tests themselves checkout different branches and do the diffs?

Pretty minor, but in the travis code, we had nosetests first, followed by yamllint.  I'm not sure if we trip up one or the other more often.

Also, nit: newline at the end of the file.

Reply:
RE: Fetch

We can't afford to check all the urls for validity so the unit tests computes the diff to `origin/master` and then only validates changed and added urls.  This is also how we detect modifications of EOL distros etc. The checkout in in the actions is stripped down more, on travis the whole of origin was available without explicitly requesting it. 

Re: Ordering

I moved the yamllint first because it takes much less time to run. So I figured to run it first to be able to give quicker feedback if it failed. Maybe I can run it in parallel with a separate check.

I'll add the new line.